src/pymedphys/_labs/paulking/profile.py

```diff
@@ -36,17 +36,17 @@
 import matplotlib.pyplot as plt
 import matplotlib.image as mpimg
 
-# from PIL import Image
 import PIL
 
-from ...libutils import get_imports
-
-# from .._level1.coreobjects import _PyMedPhysBase
-IMPORTS = get_imports(globals())
+try:
+    assert "dosepro" not in __file__
+    from ...libutils import get_imports
+    IMPORTS = get_imports(globals())

```

Comment:
It looks like this import will always fail. May as well delete these lines of code here.

Reply:
@kingrpaul I'm happy to merge this by the way, just figured it's worth letting you know if I see glaring issues...

Let me know if you want me to merge.

Reply:
@SimonBiggs much appreciated. I have things I want to do with this but it's not ready for merge yet.

Reply:
Kk :)